datasets/ms_terms/ms_terms.py

```diff
@@ -0,0 +1,111 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Microsoft Terminology Collection."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import xml.etree.ElementTree as ElementTree
+
+import datasets
+
+
+_DESCRIPTION = """\
+The Microsoft Terminology Collection can be used to develop localized versions of applications that integrate with Microsoft products.
+It can also be used to integrate Microsoft terminology into other terminology collections or serve as a base IT glossary
+for language development in the nearly 100 languages available. Terminology is provided in .tbx format, an industry standard for terminology exchange.
+"""
+
+_LICENSE = """\
+See the Microsoft Language Portal Materials License and the Microsoft Terms of Use for details.
+"""
+
+_ENTRY_ID = "entry_id"
+_TERM_SOURCE = "term_source"
+_TERM_POS = "pos"
+_TERM_DEFINITION = "definition"
+_TERM_TARGET = "term_target"
+
+_FILENAME = "MicrosoftTermCollection.tbx"
+
+
+class MsTerms(datasets.GeneratorBasedBuilder):
+    """The Microsoft Terminology Collection."""
+
+    VERSION = datasets.Version("1.0.0")
+    DEFAULT_CONFIG_NAME = "ms_terms-full"
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name=DEFAULT_CONFIG_NAME)
+    ]
```

Comment:
```suggestion
    BUILDER_CONFIGS = [
        datasets.BuilderConfig(name="ms_terms-full")
    ]
```

since there's only one config you don't need to specify the default config name :)